### PR TITLE
V1.3 DG search

### PR DESCRIPTION
**Pull request by zhing22**

### CODE REVIEW

File: docs/DeveloperGuide.md [hunk 1]
```diff
@@ -264,6 +248,55 @@ Step 5. `AddAllocCommand#execute()` calls `Group#addStudent()` to add student `A
...
+![SearchActivityDiagram](images/SearchActivityDiagram.png)
+
+<div markdown="span" class="alert alert-info">:information_source: **Note:** SearchCommandParser checks
+if command is valid. Command is invalid if user input is empty, or if user entered more or less than one parameter.
+</div>
+
+
+#### Design considerations
+
+**Aspect: How undo & redo executes:**
```
Comment (ganhongyao) [inline docs/DeveloperGuide.md]:
```suggestion
**Aspect: How search executes:**
```

File: src/main/java/seedu/address/logic/parser/AddressBookParser.java [hunk 2]
```diff
@@ -64,6 +64,7 @@ public Command parseCommand(String userInput) throws ParseException {
             return new ClearCommand();
 
         case SearchCommand.COMMAND_WORD:
+            System.out.println("argg: " + arguments);
```
Comment (ganhongyao) [inline src/main/java/seedu/address/logic/parser/AddressBookParser.java]:
```suggestion

```

Stray debug statement

**Review by ganhongyao — CHANGES_REQUESTED** [review]

File: src/main/java/seedu/address/logic/parser/AddressBookParser.java (hunk 2 above)
Comment (zhing22) [reply src/main/java/seedu/address/logic/parser/AddressBookParser.java]: omg i.. thankyou

**Review by ganhongyao — APPROVED** [review]
LGTM

**Pull request merged**